ui/components/table.js

```diff
@@ -68,26 +90,18 @@ export default function Table({
     setSelectedRowIds([])
   }, [pageIndex])
 
-  function toggleAll() {
-    setSelectedRowIds(checkedAll || indeterminate ? [] : data?.map(d => d.id))
-    setCheckedAll(!checkedAll && !indeterminate)
-    setIndeterminate(false)
-  }
-
   return (
     <div className='relative overflow-x-auto rounded-lg border border-gray-200/75'>
       {selectedRowIds.length > 0 && (
         <div className='absolute left-12 flex h-6 items-center py-4 sm:left-16'>
           <button
             type='button'
-            onClick={() => {
-              onDelete(selectedRowIds)
-            }}
+            onClick={() => onDelete()}
             className='rounded-md bg-zinc-50 px-4 py-2 text-2xs font-medium  text-red-500 hover:bg-red-100'
           >
             <div className='flex flex-row items-center'>
               <TrashIcon className='mr-1 mt-px h-3.5 w-3.5' />
-              Remoeve selected
+              Remove selected
```

Comment:
Would it be possible to move this to the right and align it with the `Remove` option of individual items?